src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -519,18 +522,20 @@ OpenTelemetry metrics release candidate was announced earlier in May, with gener
     id="trx-endpoint"
     title="Metrics with transaction/endpoint dimensions"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
 
-    The New Relic agent reports metrics (as well as spans and events) scoped to an individual transaction or endpoint.
+    The Transactions page works with New Relic agent metrics, or with metrics for HTTP services reported by OpenTelemetry. OTel instrumentation, however, does not consistently or by default report transction/endpoint metrics.
+    
+    All New Relic language agents report metrics (as well as spans and events) scoped to an individual transaction or endpoint.
 
-    Default OpenTelemetry SDK instrumentation does not consistently report such metrics. The semantic conventions for endpoint-level data, for example `http.target`, are still considered experimental.
+    Default OpenTelemetry SDK instrumentation does not consistently report such metrics. The semantic conventions for endpoint-level data, for example `http.route`, are still considered experimental.
```

Comment:
This was an important recent change to align with the OTel spec, which Jack Berg moved forward

Reply:
@rkbodenner Looks good to me!